lib/tasks/users.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+namespace :users do
+  desc 'mark all users as confirmed'
+  task confirm_all: :environment do
+    p 'Start confirming users'
+    User.all.in_batches do |group|
+      group.update_all(confirmed_at: Time.now)
```

Comment:
Can you change this to use `Time.current` to accommodate timezone issues?

Reply:
Done